packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -828,4 +919,70 @@ - (void)setMixWithOthers:(BOOL)mixWithOthers
 #endif
 }
 
+- (nullable NSNumber *)isPictureInPictureSupported:
+    (FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  if (@available(macOS 10.15, *)) {
+#if TARGET_OS_OSX
+    return @([AVPictureInPictureController isPictureInPictureSupported]);
+#else
+    return @([AVPictureInPictureController isPictureInPictureSupported] &&
+             [self doesInfoPlistSupportPictureInPicture]);
+#endif
+  } else {
+    return FALSE;
```

Comment:
This should be `@NO`. Currently you are return `null` here rather than `false`, on the Dart side, which violates the Pigeon API.